modules/web/src/main/resources/index.html

```diff
@@ -89,7 +89,7 @@
 
 <script>
 // From http://www.bootply.com/VQqzOawoYc
-// Javascript to enable link to tab
+// JavaScript to enable link to tab
```

Comment:
If this is actually from someone else's code, I could understand not applying it.

I tend to eventually chase down upstreams if people provide pointers.

Reply:
This was initially generated by docusaurus I think. It's fine fixing here.